src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts

```diff
@@ -241,8 +242,20 @@ export class ConfigurationGroupComponent implements AfterViewInit {
       }, []);
   }
 
-  formStatus(status: boolean) {
-    this.formStatusEvent.emit(status);
+  formStatus(formState: any) {
+    // find the object of changed form from groups array
+    const groupObject = this.groups.find((g: any) => g.group === formState.group);
+
+    // update the status of respected form
+    if (groupObject) {
+      groupObject.status = formState.status;
+    } else if (['Advanced Configuration', 'Security Configuration'].includes(formState.group)) {
+      this.advanceCategoriesGroup.find((g: any) => g.group === formState.group).status = formState.status;
+    }
```

Comment:
Can we write this as

```
     let groupObject = this.groups.find((g: any) => g.group === formState.group);
     // Set the status of respected tab
     if (!groupObject) {
       groupObject  = this.advanceCategoriesGroup.find((g: any) => g.group === formState.group)
     } 
     if(groupObject){
       groupObject.status = formState.status;
     }
```

I wonder if groups are invalid in Basic and Advanced both, then this if else will work?